src/vconsole/vconsole-setup.c

```diff
@@ -306,6 +311,18 @@ int main(int argc, char **argv) {
                         log_warning_errno(r, "Failed to read /proc/cmdline: %m");
         }
 
+        /* Sanitize vc_max_s */
+        if (vc_max_s == NULL)
+                vc_max = 12;
+        else {
+                r = safe_atoi(vc_max_s, &vc_max);
+                if (r != 0 || vc_max < 1 || vc_max > MAX_CONSOLES) {
+                        log_warning("VC_MAX (vconsole.vc.max) out of range, should be (1.." STRINGIFY(MAX_CONSOLES) ").");
+                        vc_max = 12;
```

Comment:
previously we looked at 15 VCs by default. You lowered this to 12. Any particular reason?